tools/ci_build/generate_nuspec_for_native_nuget.py

```diff
@@ -0,0 +1,462 @@
+                for child_file in child.iterdir():
+                    suffixes = [".dll", ".lib", ".pdb"] if include_pdbs else [".dll", ".lib"]
+                    if child_file.suffix in suffixes:
+                        files_list.append(
+                            '<file src="' + str(child_file) + '" target="runtimes/win-%s/native"/>' % cpu_arch
+                        )
+        for cpu_arch in ["x86_64", "arm64"]:
+            if child.name == get_package_name("osx", cpu_arch):
+                child = child / "lib"  # noqa: PLW2901
+                if cpu_arch == "x86_64":
+                    cpu_arch = "x64"  # noqa: PLW2901
+                for child_file in child.iterdir():
+                    # Check if the file has digits like onnxruntime.1.8.0.dylib. We can skip such things
+                    is_versioned_dylib = re.match(r".*[\.\d+]+\.dylib$", child_file.name)
+                    if child_file.is_file() and child_file.suffix == ".dylib" and not is_versioned_dylib:
+                        files_list.append(
+                            '<file src="' + str(child_file) + '" target="runtimes/osx.10.14-%s/native"/>' % cpu_arch
+                        )
+        for cpu_arch in ["x64", "aarch64"]:
+            if child.name == get_package_name("linux", cpu_arch):
+                child = child / "lib"  # noqa: PLW2901
+                if cpu_arch == "x86_64":
+                    cpu_arch = "x64"  # noqa: PLW2901
+                elif cpu_arch == "aarch64":
+                    cpu_arch = "arm64"  # noqa: PLW2901
+                for child_file in child.iterdir():
+                    if not child_file.is_file():
+                        continue
+                    if child_file.suffix == ".so":
+                        files_list.append(
+                            '<file src="' + str(child_file) + '" target="runtimes/linux-%s/native"/>' % cpu_arch
+                        )
+
+        if child.name == "onnxruntime-extensions-android":
+            for child_file in child.iterdir():
+                if child_file.suffix in [".aar"]:
+                    files_list.append('<file src="' + str(child_file) + '" target="runtimes/android/native"/>')
+
+        if child.name == "onnxruntime_extensions.xcframework":
+            files_list.append('<file src="' + str(child) + "\\**" '" target="runtimes/ios/native"/>')  # noqa: ISC001
+
+
+def generate_id(line_list, package_name):
+    line_list.append("<id>" + package_name + "</id>")
+
+
+def generate_version(line_list, package_version):
+    line_list.append("<version>" + package_version + "</version>")
+
+
+def generate_authors(line_list, authors):
+    line_list.append("<authors>" + authors + "</authors>")
+
+
+def generate_owners(line_list, owners):
+    line_list.append("<owners>" + owners + "</owners>")
+
+
+def generate_description(line_list, package_name):
+    description = ""
+
+    if package_name == "Microsoft.AI.MachineLearning":

```

Comment:
We only need to support Microsoft.ML.OnnxRuntime.Extensions don't we? I expected we did not need any logic here handling different package names/types so we can remove all that. is_cpu_package should always be true (that is for when ORT is building a GPU nuget)

Reply:
I am not very clear what does this doing?
```
        # Process props file
        source_props = os.path.join(
            args.sources_path, "csharp", "src", "Microsoft.ML.OnnxRuntime", "targets", "netstandard", "props.xml"
        )
        target_props = os.path.join(
            args.sources_path,
            "csharp",
            "src",
            "Microsoft.ML.OnnxRuntime",
            "targets",
            "netstandard",
            args.package_name + ".props",
        )
```
Can I remove them all?